### PR TITLE
Copyrights and version bump to v0.10.0 alpha

### PR DESCRIPTION
**Pull request by tbennun**
Add copyright notices, add author list, and refine the copyright holders.
This is intended to prepare DaCe for the v0.10.0 alpha pre-release.

### CODE REVIEW

**Comment by definelicht** [comment]
ugh

**Comment by tbennun** [comment]
@definelicht No worries, I'll merge it myself. It has no code changes

File: dace/runtime/runtime.vcxproj [hunk 1]
```diff
@@ -1,176 +0,0 @@
-﻿<?xml version="1.0" encoding="utf-8"?>
```
Comment (definelicht) [inline dace/runtime/runtime.vcxproj:1]: What happened here?

File: dace/runtime/runtime.vcxproj.filters [hunk 2]
```diff
@@ -1,122 +0,0 @@
-﻿<?xml version="1.0" encoding="utf-8"?>
```
Comment (definelicht) [inline dace/runtime/runtime.vcxproj.filters:1]: Ditto

File: doc/conf.py [hunk 3]
```diff
@@ -17,11 +17,11 @@
...
 
 project = 'DaCe'
-copyright = '2020, Scalable Parallel Computing Laboratory, ETH Zurich'
-author = 'Scalable Parallel Computing Laboratory, ETH Zurich'
+copyright = '2019-2020, Scalable Parallel Computing Laboratory, ETH Zurich'
+author = 'Scalable Parallel Computing Laboratory, ETH Zurich and the DaCe authors'
 
 # The full version, including alpha/beta/rc tags
-release = '0.9.5'
+release = '0.10.0a'
```
Comment (definelicht) [inline doc/conf.py:24]: Don't you want to do this separately?

**Review by definelicht — CHANGES_REQUESTED** [review]

File: dace/runtime/runtime.vcxproj (hunk 1 above)
Comment (tbennun) [reply dace/runtime/runtime.vcxproj:1]: Removed unused Visual C++ files

File: dace/runtime/runtime.vcxproj.filters (hunk 2 above)
Comment (tbennun) [reply dace/runtime/runtime.vcxproj.filters:1]: Same

File: doc/conf.py (hunk 3 above)
Comment (tbennun) [reply doc/conf.py:24]: we could

Comment (tbennun) [reply doc/conf.py:24]: @definelicht but then I'd have to open a new PR for review :)

**Review by definelicht — APPROVED** [review]

**Pull request merged**